spec/unit/core_spec.rb

```diff
@@ -123,4 +123,22 @@ class Book < ActiveFedora::Base
       it { should eq '123456w' }
     end
   end
+
+  describe "when serializing an object" do 
+    let(:book) { Book.create(title:"foo") }
+    it "should handle an empty base path" do
+      allow(ActiveFedora.fedora).to receive(:base_path).and_return("/")
+      expect(ActiveFedora::Base.find(book.id).class.name).to eq("Book")
```

Comment:
If you want to validate that the correct class is found, you should move the tests to spec/integration, because that would be an integration test not a unit test.